src/collections/_documentation/development/server/environment.md

```diff
@@ -108,45 +105,20 @@ node -v
 Volta intercepts this and automatically downloads and installs the node version in sentry's `package.json`.
 
 
-### macOS - Python (virtual environment)
+## direnv
 
-You're now ready to create a python virtual environment. Run:
+We use [direnv](https://github.com/direnv/direnv) to automate configuration and constraints. It automatically sets some helpful environment variables for you, activates your virtual environment, and does some simple state checking to guide you towards the expected development environment.
 
-```bash
-python -m pip install virtualenv
-python -m virtualenv .venv
-```
-
-And activate the virtual environment:
+To install it, run `brew install direnv`. You'll want to add this to the end of your `~/.bashrc` file:
 
 ```bash
-source .venv/bin/activate
+eval "$(direnv hook bash)"
 ```
 
-If everything worked, running `which python` should now result in something like `/Users/you/whereever-you-cloned-sentry/.venv/bin/python`.
-
-
-### macOS - Final Bootstrap
-
-The last step is to run `make bootstrap`. This will take a long time, as it basically installs sentry and all of its dependencies, starts up external services, and preps databases.
-
-{% capture __alert_content -%}
-`make bootstrap` will run `sentry upgrade`, which will prompt you to create a user. It is recommended to supply the prompts with a proper email address and password. It is also required to designate said user as a **superuser** because said user is responsible for the initial configurations.
-{%- endcapture -%}{%- include components/alert.html
-  title="Note"
-  content=__alert_content
-  level="warning"
-%}
-
+Or in your `~/.zshrc`:
 
-### macOS - direnv (optional, but recommended)
-
-We have a [direnv](https://github.com/direnv/direnv) configuration for Sentry. It automatically sets some helpful environment variables for you, automatically activates your virtual environment, and does some simple state checking to guide you towards the expected development environment.
-
-To install it, run `brew install direnv`. You'll want to add this to the end of your `~/.bashrc` file:
-
-```bash
-eval "$(direnv hook bash)"
+```zsh
```

Comment:
i just threw this in here since it was eassy and i know we'll be updating this to look at zsh in the future. can back it out if we think its noisy.

really it should just be "do what direnv says when you install it, as it already tells you to do zsh or bash correctly"

Reply:
Then I'd rather remove it but leaving the final decision up to you.